<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetriDesign</class>
 <widget class="QWidget" name="PetriDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="PetriView" name="widget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbButtons">
     <property name="minimumSize">
      <size>
       <width>475</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>475</width>
       <height>96</height>
      </size>
     </property>
     <property name="title">
      <string>_elementos</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="bt_normal">
        <property name="text">
         <string>_normal</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/cursor.ico</normaloff>:/icons/img/cursor.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bt_place">
        <property name="text">
         <string>_place</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/place.ico</normaloff>:/icons/img/place.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="bt_fplace">
        <property name="text">
         <string>_fplace</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/fplace.ico</normaloff>:/icons/img/fplace.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="bt_ttrans">
        <property name="text">
         <string>_TTrans</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/ttrans.ico</normaloff>:/icons/img/ttrans.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="bt_itrans">
        <property name="text">
         <string>_ITrans</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/imtrans.ico</normaloff>:/icons/img/imtrans.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="bt_remove">
        <property name="text">
         <string>_remover</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bt_arc">
        <property name="text">
         <string>_arc</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/arc.ico</normaloff>:/icons/img/arc.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="bt_iarc">
        <property name="text">
         <string>_Iarc</string>
        </property>
        <property name="icon">
         <iconset resource="imgs.qrc">
          <normaloff>:/icons/img/inhibitor.ico</normaloff>:/icons/img/inhibitor.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="bt_farc">
        <property name="text">
         <string>_Farc</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="cb_lock">
        <property name="text">
         <string>_travar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PetriView</class>
   <extends>QWidget</extends>
   <header>diagram/petriview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
